testplan/exporters/testing/json/__init__.py

```diff
@@ -68,3 +116,49 @@ def export(self, source):
             self.logger.exporter_info(
                 "Skipping JSON creation for empty report: %s", source.name
             )
+
+    @staticmethod
+    def split_json(data):
+        """Split a single Json into several parts."""
+
+        def split_assertions(entries, assertions):
+            """Remove assertions from report and place them in a dictionary."""
+            for entry in entries:
+                if entry.get("category") == ReportCategories.TESTCASE:
+                    if entry.get("entries"):
+                        # A testcase report should have `parent_uids`
+                        uid = _gen_unique_id(
+                            entry["parent_uids"] + [entry["name"]]
+                        )
+                        assert uid is not None and uid not in assertions
```

Comment:
I think this exception case could be better handled than using raw assert.

Reply:
Let discuss it, use readable string to replace SHA1 digest as key of assertions